libvalkeycluster/adlist.c

```diff
@@ -41,7 +41,7 @@
 hilist *listCreate(void) {
     struct hilist *list;
 
-    if ((list = hi_malloc(sizeof(*list))) == NULL)
+    if ((list = vk_malloc(sizeof(*list))) == NULL)
```

Comment:
Oh, that's a project I have missed.
hi_malloc is provided by hiredis and the renaming is originally done in the libvalkey code.
Any other suggestions for @michael-grunder

Reply:
Vulkan Memory Allocator seems to use the prefix "vmk" in their functions. I don't think we need to worry about it. It's not in out user-facing API anyway.

Reply:
@zuiderkwast you're right. And Vulkan itself uses camel case (`vkFunctionName`) in the functions.

However, it still can clash in defines, because Vulkan has `VK_*` for constants. And I'm not sure whether it can become a problem.